lib/std/json/stringify.zig

```diff
@@ -88,7 +88,7 @@ pub fn stringifyAlloc(
     allocator: Allocator,
     value: anytype,
     options: StringifyOptions,
-) error{OutOfMemory}![]u8 {
+) error{ OutOfMemory, InvalidJSON }![]u8 {
```

Comment:
Small nitpick to follow Zig style:

```suggestion
) error{ OutOfMemory, InvalidJson }![]u8 {
```

Reply:
Thank you for calling that out! My background is mostly Go and that will be a difficult habit to break, but I'll try (and I'll go update these items now!)